tests/Unit.hs

```diff
@@ -3896,18 +3910,18 @@ tests = TestList
 
 
 
-    testRegex "^[a-\\d]" []
-        ["abcde",
-         "-things",
-         "0digit",
-         "*** Failers",
-         "bcdef    "]
-        [Just ["a"],
-         Just ["-"],
-         Just ["0"],
-         Nothing,
-         Nothing]
-    ,
+    -- testRegex "^[a-\\d]" []
```

Comment:
This is the only test that fails; so far as I can tell this isn't a valid regex, so I'm unsure why this test is here. Do you have any ideas? Currently it's failing to compile the regex with the error:

```
Error in regex: invalid range in character class
```

Reply:
I didn't write this code, so I am not sure either. But since it doesn't seem valid, it's ok to remove it.